fares-data-build-tool/fdbt-netex-output
# Feature/tfn 332 update netex convertor to aurora

# Description

-   To change the functionality of our codebase so that the reference data required to generate NeTEX is sourced from Aurora and not DynamoDB.

# Testing instructions

-  This feature was tested by running the branch through our Circle CI pipeline to deploy it in our test AWS environment. Once deployed, we used the AWS console Lambda Test Event feature to put JSON files in the 'Matching' bucket so that the NeTEX generator was triggered. We then examined the generated (& validated NeTEX xml file) to check that the data we anticipated coming from Aurora was there as expected.

# Type of change


# Checklist:

-   [ See above] Able to run pr locally
-   [   ] I have made corresponding changes to the documentation if applicable
